.circleci/config.yml

```diff
@@ -50,11 +50,17 @@ jobs:
     steps:
     - checkout
     - run:
-        name: Download NDK
-        command: wget https://dl.google.com/android/repository/android-ndk-r21-linux-x86_64.zip && unzip android-ndk-r21-linux-x86_64.zip
+        name: Install rustup
+        command: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+    - run:
+        name: Install Rust 1.41
+        command: rustup install stable && rustup default stable
```

Comment:
I tried a few different version here (1.4.1, stable, the default which is 1.4.2) and none seem to work. Don't think this was broken by my changes but I could be mistaken

Reply:
Interesting. 1.41 should work, and actually I see you use here the "latest", which is probably 1.42